app/services/unreact_service.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class UnreactService < BaseService
+  include Payloadable
+
+  def call(account, status, name)
+    reaction = StatusReaction.find_by(account: account, status: status, name: name)
```

Comment:
```suggestion
  def call(account, status, emoji)
    name, domain = emoji.split('@')
    custom_emoji = CustomEmoji.find_by(shortcode: name, domain: domain)
    reaction = StatusReaction.find_by(account: account, status: status, name: name, custom_emoji: custom_emoji)
```

As of now this looks for `shortcode@domain.tld` in the database for remote emojis, making this fail as such entry cannot be found.
This suggestion basically does the same as `react_service.rb`, splitting the name and domain and also look for the custom emoji ID, making it possible to remove a reaction with a remote emoji.